dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/logging/source/AccessLogSource.java

```diff
@@ -92,12 +144,6 @@ boolean checkConfigChange(String logFormat, String loggingConfig, String logstas
                 return true;
             return false;
         }
```

Comment:
not sure I get why we have a checkConfigChange method -- wouldn't the detection of a new config just be that there's no SetterFormatter in the map when requested with a given key?